pkg/dynamiccache/internal/informers_map.go

```diff
@@ -0,0 +1,342 @@
+/*
```

Comment:
How hard do you think it will be to maintain this fork?

Does it make sense to try to upstream this?

Reply:
The changes are not pervasive so it's not untenable to maintain, but no fork is always better :)

I can definitely pursue it upstream, but I wouldn't want that to block out efforts here.